File: Linguagem C Exercicios/Vetores/MEDIA PARES.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int n,cont,soma;
    double media;

    printf("Quantos elementos vai ter o vetor? ");
    scanf("%d",&n);

    int vet[n];

    //Lendo o vetor
    for (int i=0;i<n;i++){
        printf("Digite um numero: ");
        scanf("%d",&vet[i]);
    };

    //Se tiver numeros pares, fazer a media entre eles
    soma = 0;
    cont = 0;
    media = 0;
    for (int i=0;i<n;i++){
        if (vet[i] % 2 == 0){
            soma = soma + vet[i];
            cont++;
        };
    };

    if (cont == 0){
        printf("NENHUM NUMERO PAR \n");
    }else {
        media = (double)(soma / cont);
        printf("MEDIA DOS PARES = %.1lf\n",media);
    };

    return 0;
}

File: Linguagem C#/Vetores/MediaPares.cs
using System;
using System.Globalization;

namespace MediaPares {
    class Program {
        static void Main(string[] args) {

            CultureInfo CI = CultureInfo.InvariantCulture;

            int n,cont,soma;
            double media;

            Console.Write("Quantos elementos vai ter o vetor? ");
            n = int.Parse(Console.ReadLine());

            int[] vet = new int[n];

            for (int i = 0; i < n; i++) {
                Console.Write("Digite um numero: ");
                vet[i] = int.Parse(Console.ReadLine());
            };

            //Se houver par, fazer a media
            cont = 0;
            soma = 0;
            for (int i = 0; i < n; i++) {
                if (vet[i] % 2 == 0) {
                    cont++;
                    soma = soma + vet[i];
                };
            };

            if (cont == 0) {
                Console.WriteLine("NENHUM NUMERO PAR");
            }else {
                media = (double)soma / cont;
                Console.WriteLine("MEDIA DOS PARES = " + media.ToString("F1",CI));
            }
        }
    }
}

